lib/src/widgets/chat/ds_location_message_bubble.widget.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../../enums/ds_align.enum.dart';
+import '../../models/ds_message_bubble_style.model.dart';
+import '../../themes/colors/ds_colors.theme.dart';
+import '../animations/ds_spinner_loading.widget.dart';
+import '../texts/ds_body_text.widget.dart';
+import '../utils/ds_cached_network_image_view.widget.dart';
+import 'ds_message_bubble.widget.dart';
+
+class DSLocationMessageBubble extends StatelessWidget {
+  final DSAlign align;
+  final DSMessageBubbleStyle style;
+  final String? title;
+  final String latitude;
+  final String longitude;
```

Comment:
what's happen if the latitude or longitude isn't a valid ? e.g: 'abc'

Reply:
I believe it would be better to use the latitude and longitude as a double, instead of a string.